mcc/odin/cmd/create.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Devex/spaceflight/mcc/odin/odin"
+)
+
+var instanceType, password, user string
+var size int64
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create [flags] identifier",
+	Short: "Creates a database",
+	Long:  `Creates a database, into RDS.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			log.Fatal("You should include instance identifier")
```

Comment:
I mean message says there's a missing parameter but it can fail because of too many arguments. Misleading message.